internal/exec/engine.go

```diff
@@ -72,7 +77,17 @@ func (e Engine) Run(stageName string) error {
 	e.logReport(r)
 	if err != nil && err != providers.ErrNoProvider {
 		e.Logger.Crit("failed to acquire system base config: %v", err)
-		return err
+	}
+
+	configExists, err := e.isConfigCacheExist(e.ConfigCache)
+	if err != nil {
+		e.Logger.Info("failed to read cached config: %v", err)
+	}
+
+	if !configExists {
```

Comment:
Although that's always the case in FCOS, here we should only log this if there was actually a base config (i.e. you didn't get `ErrNoProvider` higher up).